product_variant_supplierinfo/models/product_product.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2016 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import api, models, fields
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    seller_ids = fields.Many2many(
+        'product.supplierinfo',
+        compute='_compute_seller_ids')
+    seller_delay = fields.Integer(
```

Comment:
Why do you need to reimplement this? And AFAIK, if you don't put string or help attributes, they are caught from related field, so you avoid to re-translate all.